overlord/snapstate/reboot.go

```diff
@@ -115,6 +115,14 @@ func deviceModelBootBase(st *state.State, providedDeviceCtx DeviceContext) (stri
 	return bootBase, nil
 }
 
+func bootBaseSnapType(mappedTss map[snap.Type]*state.TaskSet, bootBase string) snap.Type {
```

Comment:
maybe mappedTss should be byTypeTss ? also elsewhere

Reply:
it's a nice suggestion, added